doc/source/tep/TEP0.md

```diff
@@ -199,7 +199,7 @@ start with a paragraph of the following RFC822-style headers:
 * Drivers: a list of drivers (names and e-mail addresses), in RFC822
   syntax for the To: header
 * URL: during DRAFT state, a link to the wiki place of the draft
-  (typically probably https://sourceforge.net/p/tauruslib/wiki/TEPxxx)
+  (typically probably http://www.taurus-scada.org/tep/?TEPxxx.md)
```

Comment:
If we mark the TEP0 as obsolete, maybe we shouldn't change the URLs referring to wikis (other than the header).

what do you think?